raphtory/src/db/task/task_runner.rs

```diff
@@ -244,8 +242,8 @@ impl<G: GraphViewOps, CS: ComputeState> TaskRunner<G, CS> {
         // To allow the init step to cache stuff we will copy everything from cur_local_state to prev_local_state
         prev_local_state.clone_from_slice(&cur_local_state);
 
-        while !done && self.ctx.ss() < steps && tasks.len() > 0 {
-            (done, shard_state, global_state, cur_local_state) = self.run_task_list(
+        while !_done && self.ctx.ss() < steps && tasks.len() > 0 {
```

Comment:
does this even work?

Reply:
It does - it was just to get the warning to stop complaining about a variable which is not read (even though it is)